client/gui_rpc_server_ops.cpp

```diff
@@ -775,7 +775,8 @@ void handle_get_project_config_poll(GUI_RPC_CONN& grc) {
         );
     } else {
         const char *p = grc.get_project_config_op.reply.c_str();
-        const char *q = strstr(p, "<project_config");
+        const char *q = strstr(p, "<project_config>");
+        if (!q) q = strstr(p, "<error>");
         if (!q) q = "<project_config/>\n";
```

Comment:
I don't see how there is dead code. Can you elaborate? The fact that in case of an error it no longer outputs `<project_config/>` but the error is actually a fix for a bug that was introduced when the XML declaration stripping was added in April. Before this change, the error message was output but due to the stripping it would return an empty `<project_config/>` with the error message getting lost.